pkg/apis/core/validation/shoot.go

```diff
@@ -1205,6 +1205,19 @@ func ValidateKubeAPIServer(kubeAPIServer *core.KubeAPIServerConfig, version stri
 				allErrs = append(allErrs, field.Forbidden(fldPath.Child("serviceAccountConfig", "maxTokenExpiration"), "must be at most 2160h (90d)"))
 			}
 		}
+		if len(kubeAPIServer.ServiceAccountConfig.AcceptedIssuers) > 0 {
+			issuers := sets.New[string]()
+			if kubeAPIServer.ServiceAccountConfig.Issuer != nil {
+				issuers.Insert(*kubeAPIServer.ServiceAccountConfig.Issuer)
+			}
```

Comment:
Should we print a different error for this case? Printing `field.Duplicate` can get confusing between this field and a different slice.